Azure/cc-setup-azure.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will make the next flow
+#
+# 1) Create a group in azure
+# 2) Create a VM
+# 3) Create a disk
+# 4) Attach previous disk to VM
+# 5) Set up Clear containers components
+# 6) Install and setup Kubelet, kubeadm and CRI-O.
+#
+# Requirments:
```

Comment:
I think is good practice to add a short description inside the script despite it is in REDME/docs again, maybe I just need to do this description shorter, what do you think?

Reply:
My comment was specifically about these 2 requirements being listed here **and** in the document. We need to do everything we possibly can to avoid bitrot so having the same text appear in 2 places is something we want to avoid as otherwise they are just *bound* to get out of sync at some point in the future. I don't care where we define the requirements, but please can we just have one list of them ;)